src/target/riscv/riscv.c

```diff
@@ -1699,6 +2125,7 @@ static int riscv_hit_watchpoint(struct target *target, struct watchpoint **hit_w
 	 *
 	 * OpenOCD will behave as if this function had never been implemented i.e.
 	 * report the halt to GDB with no address information. */
+	LOG_TARGET_DEBUG(target, "Not hit address=%" TARGET_PRIxADDR, mem_addr);
```

Comment:
I'd recommend to rephrase the debug message for more clarity.

Prefix `0x` should be used for hexadecimal numbers.

Also it would be good to say in the message if this was load or store. In any case, the information about the type of the access (`is_load`) will be needed in this function anyway to fix [the above TODO](https://github.com/riscv-collab/riscv-openocd/pull/1029/files#diff-b4aa16f9e42cb8f0934baa7c8e0ec9c70a369bef98b99b26ae2e896c8aa95d6aR2112).

```suggestion
	LOG_TARGET_DEBUG(target, "No watchpoint found that would cover %s 0x%" TARGET_PRIxADDR, is_load ? "load from" : "store to" , mem_addr);
```